holonote/annotate/display.py

```diff
@@ -481,7 +487,9 @@ def overlay(self, indicators=True, editor=True) -> hv.Overlay:
 
     def static_indicators(self, **events):
         fields_labels = self.annotator.all_fields
-        region_labels = [k for k in self.data.columns if k not in fields_labels]
+        region_labels = [
+            k for k in self.data.columns if k not in fields_labels and k != "__selected__"
```

Comment:
Why was this removed?

Reply:
Wasn't removed; just added `and k != "__selected__"

Reply:
Why was `__selected__` removed?

Reply:
I forget how I reasoned through this, but without this change, upon clicking on an existing dot:

```

WARNING:param.AnnotationDisplay.static_indicators: Callable raised "ValueError("List parameter 'PipelineMeta.kdims' length must be between 2 and 2 (inclusive), not 3.")".
Invoked as AnnotationDisplay.static_indicators(_count=0, selected_indices=['2027963e629542a79a82a2ee2e574d54'])
WARNING:param.AnnotationDisplay.static_indicators: Callable raised "ValueError("List parameter 'PipelineMeta.kdims' length must be between 2 and 2 (inclusive), not 3.")".
Invoked as AnnotationDisplay.static_indicators(_count=0, selected_indices=['2027963e629542a79a82a2ee2e574d54'])
Traceback (most recent call last):
  File "/Users/ahuang/repos/holoviews/holoviews/plotting/util.py", line 293, in get_plot_frame
    return map_obj[key]
  File "/Users/ahuang/repos/holoviews/holoviews/core/spaces.py", line 1216, in __getitem__
    val = self._execute_callback(*tuple_key)
  File "/Users/ahuang/repos/holoviews/holoviews/core/spaces.py", line 983, in _execute_callback
    retval = self.callback(*args, **kwargs)
  File "/Users/ahuang/repos/holoviews/holoviews/core/spaces.py", line 552, in __call__
    return self.callable()
  File "/Users/ahuang/repos/holoviews/holoviews/util/__init__.py", line 1038, in dynamic_operation
    key, obj = resolve(key, kwargs)
  File "/Users/ahuang/repos/holoviews/holoviews/util/__init__.py", line 1027, in resolve
    return key, map_obj[key]
  File "/Users/ahuang/repos/holoviews/holoviews/core/spaces.py", line 1216, in __getitem__
    val = self._execute_callback(*tuple_key)
  File "/Users/ahuang/repos/holoviews/holoviews/core/spaces.py", line 983, in _execute_callback
    retval = self.callback(*args, **kwargs)
  File "/Users/ahuang/repos/holoviews/holoviews/core/spaces.py", line 552, in __call__
    return self.callable()
  File "/Users/ahuang/repos/holoviews/holoviews/core/overlay.py", line 34, in dynamic_mul
    element = other[args]
  File "/Users/ahuang/repos/holoviews/holoviews/core/spaces.py", line 1216, in __getitem__
    val = self._execute_callback(*tuple_key)
  File "/Users/ahuang/repos/holoviews/holoviews/core/spaces.py", line 983, in _execute_callback
    retval = self.callback(*args, **kwargs)
  File "/Users/ahuang/repos/holoviews/holoviews/core/spaces.py", line 552, in __call__
    return self.callable()
  File "/Users/ahuang/repos/holoviews/holoviews/core/spaces.py", line 197, in dynamic_mul
    self_el = self.select(HoloMap, **key_map) if self.kdims else self[()]
  File "/Users/ahuang/repos/holoviews/holoviews/core/spaces.py", line 1216, in __getitem__
    val = self._execute_callback(*tuple_key)
  File "/Users/ahuang/repos/holoviews/holoviews/core/spaces.py", line 983, in _execute_callback
    retval = self.callback(*args, **kwargs)
  File "/Users/ahuang/repos/holoviews/holoviews/core/spaces.py", line 552, in __call__
    return self.callable()
  File "/Users/ahuang/repos/holoviews/holoviews/core/overlay.py", line 34, in dynamic_mul
    element = other[args]
  File "/Users/ahuang/repos/holoviews/holoviews/core/spaces.py", line 1216, in __getitem__
    val = self._execute_callback(*tuple_key)
  File "/Users/ahuang/repos/holoviews/holoviews/core/spaces.py", line 983, in _execute_callback
    retval = self.callback(*args, **kwargs)
  File "/Users/ahuang/repos/holoviews/holoviews/core/spaces.py", line 581, in __call__
    ret = self.callable(*args, **kwargs)
  File "/Users/ahuang/repos/holonote/holonote/annotate/display.py", line 511, in static_indicators
    indicator = Indicator.points_2d(**indicator_kwargs)
  File "/Users/ahuang/repos/holonote/holonote/annotate/display.py", line 181, in points_2d
    element = hv.Points(data, kdims=region_labels, vdims=vdims)
  File "/Users/ahuang/repos/holoviews/holoviews/element/selection.py", line 25, in __init__
    super().__init__(*args, **kwargs)
  File "/Users/ahuang/repos/holoviews/holoviews/core/data/__init__.py", line 332, in __init__
    super().__init__(data, **dict(kwargs, **dict(dims, **extra_kws)))
  File "/Users/ahuang/repos/holoviews/holoviews/core/dimension.py", line 839, in __init__
    super().__init__(data, **params)
  File "/Users/ahuang/repos/holoviews/holoviews/core/dimension.py", line 500, in __init__
    super().__init__(**params)
  File "/Users/ahuang/miniconda3/envs/holonote/lib/python3.10/site-packages/param/parameterized.py", line 4148, in __init__
    refs, deps = self.param._setup_params(**params)
  File "/Users/ahuang/miniconda3/envs/holonote/lib/python3.10/site-packages/param/parameterized.py", line 1678, in override_initialization
    ret = fn(self_, *args, **kw)
  File "/Users/ahuang/miniconda3/envs/holonote/lib/python3.10/site-packages/param/parameterized.py", line 1962, in _setup_params
    setattr(self, name, val)
  File "/Users/ahuang/miniconda3/envs/holonote/lib/python3.10/site-packages/param/parameterized.py", line 527, in _f
    return f(self, obj, val)
  File "/Users/ahuang/miniconda3/envs/holonote/lib/python3.10/site-packages/param/parameterized.py", line 1490, in __set__
    self._validate(val)
  File "/Users/ahuang/miniconda3/envs/holonote/lib/python3.10/site-packages/param/parameters.py", line 2520, in _validate
    self._validate_bounds(val, self.bounds)
  File "/Users/ahuang/miniconda3/envs/holonote/lib/python3.10/site-packages/param/parameters.py", line 2531, in _validate_bounds
    raise ValueError(
ValueError: List parameter 'PipelineMeta.kdims' length must be between 2 and 2 (inclusive), not 3.
```